modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java

```diff
@@ -73,6 +73,9 @@ public Map<String, Object> getParameters(
 			"editingLocale", localeJSONObject
 		).put(
 			"localizedObjectField", localizedObjectField
+		).put(
+			"portletNamespace",
+			ddmFormFieldRenderingContext.getPortletNamespace()
```

Comment:
Hi @larissacribeiro, I can see that `DocumentLibrary.es.js` can get the `portletNamespace` like this:

`const {portletNamespace} = useConfig();`

Could you try this in the context of rich text?

Reply:
Hi @igor-franca, I just did, it works both on builder and submission, I've replaced it. Thanks!